dashboard/app/views/levels/_standalone_video.html.haml

```diff
@@ -0,0 +1,18 @@
+:javascript
+  function getResult() {
+    return {
+      response: 'ok',
+      result: true
+    };
+  }
+
+.standalone-video
+  - video = @level.specified_autoplay_video
+  - src = "/videos/embed/#{video.youtube_code}?rel=0"
```

Comment:
It seems that `embed_url` is what you're looking for. Would `Video.embed_url(video.youtube_code)` work?

Or if we wanted to clean it up further, you could add a helper method to `video.rb`, something like:

```
def embed_youtube
  self.embed_url(youtube_code)
end
```

then you could use `video.embed_youtube`.